src/examples/java/com/amazonaws/crypto/examples/keyring/awskms/ActSimilarToAwsKmsMasterKeyProvider.java

```diff
@@ -84,19 +98,41 @@ public static void run(final AwsKmsCmkId awsKmsCmk, final List<AwsKmsCmkId> awsK
         //
         // This keyring reproduces the encryption behavior of the AWS KMS master key provider.
         //
-        // The AWS KMS keyring requires that you explicitly identify the CMK
+        // The AWS KMS symmetric multi-CMK keyring requires that you explicitly identify the CMK
         // that you want the keyring to use to generate the data key.
-        final Keyring cmkKeyring = StandardKeyrings.awsKmsBuilder()
-                .generatorKeyId(awsKmsCmk)
-                .keyIds(awsKmsAdditionalCmks)
+        final Keyring cmkKeyring = StandardKeyrings.awsKmsSymmetricMultiCmkBuilder()
+                .generator(awsKmsCmk)
+                .keyNames(awsKmsAdditionalCmks)
                 .build();
 
-        // Create an AWS KMS discovery keyring that will attempt to decrypt
-        // any data keys that were encrypted under an AWS KMS CMK.
-        final Keyring discoveryKeyring = StandardKeyrings.awsKmsDiscoveryBuilder().build();
-
-        // Combine the CMK and discovery keyrings
-        // to create a keyring that behaves like an AWS KMS master key provider.
+        // Create an AWS KMS symmetric multi-region discovery keyring that will attempt to decrypt
+        // any data keys that were encrypted under an AWS KMS CMK in a specific list of AWS regions.
+        //
+        // Please note that the multi-region discovery keyring requires the specific list of AWS regions
+        // it may communicate with.
+        //
+        // In production, if you need a keyring that attempts decryption in all AWS regions,
+        // you should call a service/API to get an updated list of AWS regions.
+        // This will prevent any AWS SDK-derived region-lists from potentially becoming stale over time.
```

Comment:
This is meant to convey that, if the customer wants to automatically populate the required list, they SHOULD NOT do so by calling the AWS regions enum list in the AWS SDK since it will become stale over time.

Reply:
Updated language to clarify this.